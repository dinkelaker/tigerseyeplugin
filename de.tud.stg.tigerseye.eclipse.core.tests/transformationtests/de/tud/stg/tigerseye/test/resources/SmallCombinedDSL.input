
	int a = ∑ 1, 2
	int b = ∑ 2, 3
	
	println a
	println b
	
    SELECT "id", "age", "haircolor" FROM "students"
	
