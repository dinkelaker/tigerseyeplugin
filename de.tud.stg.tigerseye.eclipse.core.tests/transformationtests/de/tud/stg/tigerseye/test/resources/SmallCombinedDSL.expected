

int a= sum__p0([1,2] as int[])

int b= sum__p0([2,3] as int[])

println a

println b

	selectFrom(
	[
		"id",
 		"age",
 		"haircolor"
	] as String[],
	[
		"students"
	] as String[])

